pages/views/frontend.py

```diff
@@ -56,63 +56,32 @@ def frontend(request):
         request.session.set_expiry(0)
         
         # grab client cert form the request create user hash, ignore if no cert info is found in request
-        
         try:
             certInfo = getCert(request)
+            cftsUser = getOrCreateUser(request, certInfo)
+
+            if cftsUser == None or not request.user.is_authenticated:
+                return redirect("/login")
+            elif cftsUser.update_info == True:
+                return redirect("/user-info")
+            elif cftsUser.banned == True:
+                if date.today() >= cftsUser.banned_until:
+                    cftsUser.banned=False
+                    cftsUser.save()
+
+            nets = getDestinationNetworks(request, cftsUser)
+
+            rc = {'networks': nets, 'resources': resources, 'user': cftsUser, 'browser': browser}
 
             # empty cert, IIS is set to ignore certs
```

Comment:
- we don't really need the if statements for certInfo['status'] being empty or bugged
- those rc variables are no longer used

Reply:
Done